File: java/common-tools/tool-encrypt/src/main/java/cn/edu/ntu/boot/encrypt/ToolEncryptApplication.java
package cn.edu.ntu.boot.encrypt;

import common.swagger.annotation.EnableSwagger;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

/**
 * @author zack <br>
 * @create 2020-05-03 21:23 <br>
 */
@EnableSwagger
@SpringBootApplication
public class ToolEncryptApplication {

    public static void main(String[] args) {
        SpringApplication.run(ToolEncryptApplication.class, args);
    }
}

File: backend/j-tools/tool-encrypt/src/main/java/io/github/alice52/ToolEncryptApplication.kt
package io.github.alice52

import common.swagger.annotation.EnableSwagger
import io.github.alice52.common.inject.annotation.SimpleBootApplication
import org.springframework.boot.runApplication

@EnableSwagger
@SimpleBootApplication
open class ToolEncryptApplication

fun main(vararg args: String) {

    runApplication<ToolEncryptApplication>(*args)
}